utils/log.go

```diff
@@ -0,0 +1,27 @@
+package utils
+
+import (
+	"os"
+	"strings"
+	"testing"
+)
+
+func GetLogFileName(t *testing.T, label string) string {
```

Comment:
- Does it need to be exported?
- No need to prefix with `Get`: https://go.dev/doc/effective_go#Getters
```suggestion
func logFileName(t *testing.T, label string) string {
```

Reply:
Before we wrote log files in the calling module, so we could have used this function to get the standardised file name. But I believe we moved all the log file writing into this module, so exporting this function might not be needed anymore. I'll verify this and change if that is the case.